src/main/java/org/mitre/synthea/engine/Logic.java

```diff
@@ -372,9 +377,28 @@ public boolean test(Person person, long time) {
   public static class ActiveCondition extends ActiveLogic {
     @Override
     public boolean test(Person person, long time) {
+      if (this.valueSet != null) {
+
+        Set<String> presentCodes = new HashSet<>(person.record.present.keySet());
```

Comment:
That is highly unlikely due the patterns used in the different code systems we use.

Reply:
It's even impossible between ICD-9, 10, LOINC, and the other systems. But between RxNorm and SNOMED I can't say that it's impossible. And the reason I'm bringing it up here is that a value set could include codes from other systems and that increases the possibility of collisions.

Plus if a collision does occur it's going to cause unexpected and inconsistent behavior in a module which won't necessarily be found quickly or easily. I'd rather try to prevent those kinds of issues before they happen.